python/pacemaker/_cts/environment.py

```diff
@@ -329,7 +330,7 @@ def _parse_args(self, argv):
         if not argv:
             argv = sys.argv[1:]
 
-        parser = argparse.ArgumentParser(epilog="python %s -g virt1 -r --stonith ssh --schema pacemaker-2.0 500" % sys.argv[0])
+        parser = argparse.ArgumentParser(epilog="%s %s -g virt1 -r --stonith ssh --schema pacemaker-2.0 500" % (BuildOptions.PYTHON, sys.argv[0]))
```

Comment:
After all that, it occurs to me that the command files are executable, and don't need to be explicitly called with python. Fine to leave as is, or to leave the new variable but drop its use here

Reply:
I've dropped the patch to add BuildOptions (why add stuff if we don't need it?) and removed the python bit from this string.